awx/ui_next/src/screens/Credential/shared/CredentialForm.jsx

```diff
@@ -84,6 +84,16 @@ function CredentialFormFields({
     [setFieldValue]
   );
 
+  function verifyGalaxyCredential(credentialId) {
+    if (credentialId) {
+      return (
+        credentialTypes[credentialId].name ===
```

Comment:
Rather than doing a string match on the name, I think it'd be better to match on the `kind` field.

```
credentialTypes[credentialId].kind === 'galaxy`
```

This is much less likely to change in the future and while I don't think `Ansible Galaxy/Automation Hub API Token` would ever get translated, if it did this logic would break.